scripts/gha/ui_testing/uitest_android/gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
This should have a copyright and license notice. See the gradle files in our integration_tests, but please do update the copyright year.